library/src/options.cxx

```diff
@@ -137,9 +137,11 @@ options::options()
   this->Internals->init("render.effect.ambient-occlusion", false);
   this->Internals->init("render.effect.tone-mapping", false);
 
+  this->Internals->init("render.hdri.file", std::string());
+  this->Internals->init("render.hdri.ambient", false);
   this->Internals->init("render.background.color", std::vector<double>{ 0.2, 0.2, 0.2 });
-  this->Internals->init(
-    "render.background.hdri", std::string()); // XXX This overrides background.color
+  this->Internals->init("render.background.hdri", std::string());
```

Comment:
no warn about it will be deprecated here?

Reply:
I cant warn but I should not add it if F3D_NO_DEPRECATED is defined. good find.

@Meakk do you think we should include deprecation logic in f3d::options directly ?

Reply:
Yes I think so. In the `set()` function, we are already testing if the option is valid. We could test if it's deprecated too.

Reply:
that seems doable but the warnings will be spammy

Reply:
Done, the output looks like this:

```
Option render.background.hdri is deprecated
Option render.background.hdri is deprecated
Option render.background.hdri is deprecated
--hdri option is deprecated, please use --hdri-file, --hdri-ambient and --hdri-skybox instead.
```

Reply:
Why is the option set three times?

Reply:
cause we parse the options (and call the getAsStringRef) three times:
 - command line args
 - global config
 - regex config

Reply:
imo it's important so even if it's a bit spammy it's needed and can be acceptable like that, wdyt @Meakk ?

Reply:
yeah that's fine